modules/gltf/gltf_document.cpp

```diff
@@ -3234,17 +3235,24 @@ Error GLTFDocument::_parse_textures(Ref<GLTFState> p_state) {
 		}
 		p_state->textures.push_back(t);
 
+		Ref<Texture> tex;
+
 		// Create and cache the texture used in the engine
-		Ref<ImageTexture> imgTex;
-		imgTex.instance();
-		imgTex->create_from_image(p_state->images[t->get_src_image()]);
+		if (p_state->external_images_paths.has(t->get_src_image())) {
+			tex = ResourceLoader::load(p_state->external_images_paths[t->get_src_image()]);
+		} else {
+			Ref<ImageTexture> imgTex;
+			imgTex.instance();
+			imgTex->create_from_image(p_state->images[t->get_src_image()]);
+			tex = imgTex;
```

Comment:
```suggestion
			Ref<ImageTexture> img_tex;
			img_tex.instance();
			img_tex->create_from_image(p_state->images[t->get_src_image()]);
			tex = img_tex;
```
While we're here, let's fix the style here